src/psycopmlutils/timeseriesflattener/flattened_dataset.py

```diff
@@ -73,18 +76,19 @@ def __init__(
         # in self.df
         self.df_aggregating = self.pred_times_with_uuid
         self.df = self.pred_times_with_uuid
+        self.loaders_catalogue = data_loaders
 
-    def add_predictors_from_list_of_argument_dictionaries(
+    def add_temporal_predictors_from_list_of_argument_dictionaries(
         self,
         predictor_list: List[Dict[str, str]],
-        predictor_dfs_dict: Dict[str, DataFrame],
+        predictor_dfs_dict: Dict[str, DataFrame] = None,
         resolve_multiple_fn_dict: Optional[Dict[str, Callable]] = None,
     ):
         """Add predictors to the flattened dataframe from a list.
 
         Args:
             predictor_list (List[Dict[str, str]]): A list of dictionaries describing the prediction_features you'd like to generate.
-            predictor_dfs (Dict[str, DataFrame]): A dictionary mapping the predictor_df in predictor_list to DataFrame objects.
+            predictor_dfs_dict (Dict[str, DataFrame], optional): A dictionary mapping the predictor_df in predictor_list to DataFrame objects. Optional, can be replaced by adding callables returning dfs to @data_loaders. Defaults to None.
```

Comment:
remove type hint and split line.

Type hint does not allow for None, either update type hint or change to {}
```suggestion
            predictor_dfs (Dict[str, DataFrame], optional): A dictionary mapping the predictor_df in predictor_list to DataFrame objects. Defaults to None, in which case it uses [...]. Can be replaced by adding callables  returning dfs to @data_loaders. 
```
Not sure what is meant by:
```
Can be replaced by adding callables  returning dfs to @data_loaders. Defaults to None.
```
Generally not sure what the default is.